.goreleaser.yaml

```diff
@@ -1,16 +1,88 @@
 before:
   hooks:
     - go mod tidy
+
+env:
+  - GO111MODULE=on
+  - CGO_ENABLED=0
+
+#gomod:
+#  proxy: true
+
 builds:
-  - env:
-      - CGO_ENABLED=0
+  - binary: addlicense
     goos:
       - linux
       - windows
       - darwin
     goarch:
       - amd64
       - arm64
+#signs:
+#  - cmd: cosign
+#    env:
+#      - COSIGN_EXPERIMENTAL=1
+#    certificate: '{{ trimsuffix .Env.artifact ".txt" }}.pem'
+#    args:
+#      - sign-blob
+#      - '--oidc-issuer=https://token.actions.githubusercontent.com'
+#      - '--output-certificate=${certificate}'
+#      - '--output-signature=${signature}'
+#      - '${artifact}'
+#docker_signs:
+#  - cmd: cosign
+#    env:
+#      - COSIGN_EXPERIMENTAL=1
+#    artifacts: manifests
+#    args:
+#      - 'sign'
+#      - '--oidc-issuer=https://token.actions.githubusercontent.com'
+#      - '${artifact}'
+dockers:
+  - image_templates:
+      - 'google/addlicense:{{ .Tag }}-amd64'
+      - 'ghcr.io/google/addlicense:{{ .Tag }}-amd64'
```

Comment:
this is really unfortunate that this is hardcoded.  That makes it a lot harder to test this in forks, unless I'm missing something.  For example, your https://github.com/developer-guy/addlicense/releases/tag/v0.999.0 tag demonstrates creating the release artifacts, but doesn't demonstrate publishing the docker image.  That something we were able to do with the actions based approach.

It doesn't look like goreleaser's [name templates](https://goreleaser.com/customization/templates/) supports the GitHub repo owner, and [custom variables](https://goreleaser.com/customization/templates/#custom-variables) are a "pro" feature.  

We need some way to test image publication to GHCR before making changes live.

Reply:
btw, we can use those variables as an environment variable to pass the registry and the image name to the GoReleaser by using the same syntax, can't we?

```shell
${{ env.REGISTRY }}/${{ env.IMAGE_NAME }
```

Reply:
hello @willnorris, I've updated the please see. We need environment variables named like this `REGISTRY` and `IMAGE_NAME`.

* https://github.com/google/addlicense/pull/101/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR48-R49

* https://github.com/google/addlicense/pull/101/files#diff-7326b55c062b0f46fe9e39aace0a25f4515cf206040fb91a6fd2cae839f5e826R20